build/dependencies.props

```diff
@@ -1,6 +1,7 @@
 ﻿<Project>
   <PropertyGroup>
-    <CommandLineUtilsVersion>1.2.0-*</CommandLineUtilsVersion>
+    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <CommandLineUtilsVersion>2.0.0-*</CommandLineUtilsVersion>
```

Comment:
Nit: this can be the same as AspNetCoreVersion as the M.Ext.CommandLineUtils.Sources package revs with the rest of aspnet.